### DISCUSSION TITLE
why this project named theia?

### DISCUSSION THREAD

**Discussion by longfei-zhang** [eclipse-theia/theia#12528] [General] [answered]
Theia is the goddess in Greek mythology. I want to know why this project named theia? 
Is there any special meaning?

**Comment by MatthewKhouzam** [comment, marked as answer]
Excellent question! Eclipse projects are named after heavenly bodies. e.g. Eclipse Sirius, Eclipse itself, and Eclipse Orbit.

The wiki article here https://en.wikipedia.org/wiki/Theia_(planet) describes Theia as a 9th planet which is smaller than mars that collides with the earth (Discrupting it) and making the moon, the source of eclipses. :)  We thought this would be appropriate as a name. 

The titan Theia in greek mythology had many children too. Much like Theia is designed with RCPs and derivation in mind.

All that together as far as I know was the reasoning. I'm sure others can explain more eloquently.

edit: Also, check out https://github.com/eclipse-theia/theia/issues/454 for more info.

**Comment by longfei-zhang** [comment]
wow thx a lot.
